src/client/telemetry/index.ts

```diff
@@ -110,14 +113,21 @@ export function sendTelemetryEvent<P extends IEventNamePropertyMapping, E extend
     let eventNameSent = eventName as string;
 
     if (ex) {
-        // When sending telemetry events for exceptions no need to send custom properties.
-        // Else we have to review all properties every time as part of GDPR.
-        // Assume we have 10 events all with their own properties.
-        // As we have errors for each event, those properties are treated as new data items.
-        // Hence they need to be classified as part of the GDPR process, and thats unnecessary and onerous.
-        eventNameSent = 'ERROR';
-        customProperties = { originalEventName: eventName as string, stackTrace: serializeStackTrace(ex) };
-        reporter.sendTelemetryErrorEvent(eventNameSent, customProperties, measures, []);
+        if (!sendOriginalEventWithErrors) {
+            // When sending telemetry events for exceptions no need to send custom properties.
+            // Else we have to review all properties every time as part of GDPR.
+            // Assume we have 10 events all with their own properties.
+            // As we have errors for each event, those properties are treated as new data items.
+            // Hence they need to be classified as part of the GDPR process, and thats unnecessary and onerous.
+            eventNameSent = 'ERROR';
+            customProperties = { originalEventName: eventName as string, stackTrace: serializeStackTrace(ex) };
+            reporter.sendTelemetryErrorEvent(eventNameSent, customProperties, measures, []);
+        } else {
+            // Include a property failed, to indicate there are errors.
+            // Lets pay the price for better data.
+            customProperties = { failed: 'true', stackTrace: serializeStackTrace(ex) };
+            reporter.sendTelemetryEvent(eventNameSent, customProperties, measures);
```

Comment:
This way we can keep using the same envet.
`NotebookStart` & look for properties whether it failed or not.
Easier than looking for other events `ERROR` & then filtering the properties to check when a particular telemetry point failed.

Hopefully @greazer  will be able to just accept the new telemetry proeprties as they arrive. Worst case we have 2 new properties for each event. & the names are generic enough to be accepted easily.